gui/package.json

```diff
@@ -12,12 +12,14 @@
     "test:ui:watch": "vitest"
   },
   "dependencies": {
+    "@headlessui/react": "^1.7.13",
     "@heroicons/react": "^2.0.15",
     "@hookform/resolvers": "^2.9.11",
     "@tailwindcss/typography": "^0.5.9",
     "@tauri-apps/api": "^1.2.0",
     "@tauri-apps/cli": "^1.2.2",
     "@vitejs/plugin-react": "^3.0.0",
+    "add": "^2.0.6",
```

Comment:
@naps62, what's this for?

Reply:
what the hell. maybe I did `yarn add add` by mistake at some point? xD